src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -3490,7 +3491,7 @@ public override void PostVisit(DottedNameNode node)
 
                         if (!leftType.ToRecord().TryGetType(property.InvariantName, out typeRhs))
                         {
-                            SetDottedNameError(node, TexlStrings.ErrInvalidName);
+                            SetDottedNameError(node, TexlStrings.ErrInvalidName, firstNodeLhs.Ident.Name.ToString());
```

Comment:
the invalid name here is Property.InvariantName

Reply:
Fixed